src/openms/include/OpenMS/ANALYSIS/QUANTITATION/AbsoluteQuantitationMethod.h

```diff
@@ -106,6 +106,46 @@ namespace OpenMS
     String transformation_model_; ///< transformation model
     Param transformation_model_params_; ///< transformation model parameters
   };
+
+  inline OPENMS_DLLAPI bool operator==(const AbsoluteQuantitationMethod& lhs, const AbsoluteQuantitationMethod& rhs)
+  {
+    if (
+      lhs.getComponentName() != rhs.getComponentName() ||
+      lhs.getFeatureName() != rhs.getFeatureName() ||
+      lhs.getISName() != rhs.getISName() ||
+      lhs.getLLOD() != rhs.getLLOD() ||
+      lhs.getULOD() != rhs.getULOD() ||
+      lhs.getLLOQ() != rhs.getLLOQ() ||
+      lhs.getULOQ() != rhs.getULOQ() ||
+      lhs.getNPoints() != rhs.getNPoints() ||
+      lhs.getCorrelationCoefficient() != rhs.getCorrelationCoefficient() ||
+      lhs.getConcentrationUnits() != rhs.getConcentrationUnits() ||
+      lhs.getTransformationModel() != rhs.getTransformationModel()
+    )
+    {
+      return false;
+    }
+    // compare params
+    const Param lp = lhs.getTransformationModelParams();
+    const Param rp = rhs.getTransformationModelParams();
+    if (lp.size() != rp.size())
```

Comment:
I think you could just replace this with 
`if (lp == rp)`
right?

Reply:
or put the whole param into the tuple of the std::tie ... or do I miss something?